### PR TITLE
AMP-39 image sizes

### PR DESCRIPTION
**Pull request by wajerrr**

### CODE REVIEW

File: src/config/images.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import baseConfig from '../server/config/economist';
```
Comment (dosyara) [inline src/config/images.js]: maybe `src/styles` would be a better place for a file?

File: src/app/components/article/article-main-image.js [hunk 2]
```diff
@@ -28,6 +29,8 @@ const ArticleMainImage = ({ src, width, height }) => {
         src={src}
         width={width}
         height={height}
+        srcset={getSrcset(src)}
```
Comment (dosyara) [inline src/app/components/article/article-main-image.js]: I believe now it make sense to have a `amp-img` component which adds `srcset` and `sizes`.

File: src/app/components/image/image.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+import spacings from '../../styles/spacings';
+
+const defaultServicePath = '/sites/default/files/imagecache/';
+
+const getSrcset = (
+  imgURL,
+  baseUrl = baseConfig.domain,
+  servicePath = defaultServicePath
+) => {
+  const imgId = imgURL.split(baseUrl)[1];
```
Comment (dosyara) [inline src/app/components/image/image.js]: Let's guard this logic and make it work only for images from e.com

Comment (wajerrr) [reply src/app/components/image/image.js]: done

**Pull request merged**